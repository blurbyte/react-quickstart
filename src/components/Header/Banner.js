import styled from 'styled-components';

const Banner = styled.header`
  position: fixed;
  display: flex;
  align-items: center;
  justify-content: space-between;
  padding: 0 6.4rem;
  top: 0;
  left: 0;
  width: 100%;
  border-bottom: 1px solid #908e89;
  height: 6.4rem;
  box-shadow: 0 4px 4px -1px rgba(40,40,40,0.1);
  background: #fff;
`;

export default Banner;
