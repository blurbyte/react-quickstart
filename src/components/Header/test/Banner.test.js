import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Banner from '../Banner';

describe('<Banner />', () => {
  it('should render correctly', () => {
    // component init
    const component = renderer.create(
      <Banner />
    );

    expect(component.toJSON()).toMatchStyledComponentsSnapshot();
  });
});
