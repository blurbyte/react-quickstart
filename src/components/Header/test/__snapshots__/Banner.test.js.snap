// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Banner /> should render correctly 1`] = `
.bEHrcd {
  position: fixed;
  display: -webkit-box;
  display: -moz-box;
  display: -ms-flexbox;
  display: -webkit-flex;
  display: flex;
  align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  -webkit-align-items: center;
  justify-content: space-between;
  -webkit-box-pack: justify;
  -ms-flex-pack: justify;
  -webkit-justify-content: space-between;
  padding: 0 6.4rem;
  top: 0;
  left: 0;
  width: 100%;
  border-bottom: 1px solid #908e89;
  height: 6.4rem;
  box-shadow: 0 4px 4px -1px rgba(40,40,40,0.1);
  background: #fff;
}

<header
  className="bEHrcd"
/>
`;
